build/rpms/cri-tools.spec

```diff
@@ -7,15 +7,13 @@ Summary: Container Runtime Interface tools
 URL: https://kubernetes.io
 
 %description
-Binaries to interface with the container runtime.
```

Comment:
Opened https://github.com/kubernetes-incubator/cri-tools/issues/330 for this

Reply:
This is a typo fix. We only provided crictl since that's the specific file we download from the release.

Were you expecting us to package critest as well? 

Take a look at line [18](https://github.com/kubernetes/kubernetes/pull/65281/files#diff-4993adbbb2bb3a1a71b4a3bd241063f4L18) -- we only ever installed crictl. 

Overall I think this is a cleaner implementation that uses bazel built-ins instead of side stepping them and needing to hardcode the version.